frontend/src/utils/random.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Return a random item from `array`.
+ */
+export function getRandom(array: Array<any>): any {
```

Comment:
A use of templates is more relevant here
```suggestion
export function getRandom<T>(array: Array<T>): T {
```